tests/test_shardformer/test_model/test_shard_deepseek_skip.py

```diff
@@ -0,0 +1,232 @@
+                atol=atol,
+                rtol=rtol,
+                dim=1,
+                verbose=False,
+            )
+        except Exception as e:
+            rank = dist.get_rank()
+            print(f"{rank=}, Failed config: {test_config}")
+            raise e
+
+    torch.cuda.empty_cache()
+
+
+@parameterize(
+    "test_config",
+    [
+        # {
+        #     "tp_size": 1,
+        #     "pp_size": 1,
+        #     "num_microbatches": 2,
+        #     "ep_size": 2,
+        #     "zero_stage": 0,
+        #     "overlap_communication": False,
+        #     "precision": "fp16",
+        # },  # [dp(4)] + [moe_dp(4)]
+        # {
+        #     "tp_size": 1,
+        #     "pp_size": 2,
+        #     "num_microbatches": 2,
+        #     "ep_size": 2,
+        #     "zero_stage": 1,
+        #     "overlap_communication": False,
+        #     "precision": "fp32",
+        # },  # [dp(2) + pp(2)] + [moe_pp(2)]
+        # {
+        #     "tp_size": 1,
+        #     "pp_size": 2,
+        #     "ep_size": 2,
+        #     "num_microbatches": 2,
+        #     "zero_stage": 1,
+        #     "overlap_communication": False,
+        #     "precision": "fp16",
+        #     "initial_scale": 1,
+        #     "find_unused_parameters": True,
+        # },  # [pp(2) + tp(2)] + [pp(2), replicate(2)] pass
+        {  # Ulysess + Flash attention
+            "tp_size": 1,
+            "pp_size": 1,
+            "sp_size": 2,
+            "ep_size": 2,
+            "enable_sequence_parallelism": True,
+            "sequence_parallelism_mode": "all_to_all",
+            "zero_stage": 1,
+            "overlap_communication": False,
+            "precision": "fp16",
+            "initial_scale": 1,
+            "find_unused_parameters": True,
+        },
+        # {
+        #     "tp_size": 1,
+        #     "pp_size": 1,
+        #     "ep_size": 2,
+        #     "zero_stage": 0,
+        #     "overlap_communication": False,
+        #     "precision": "fp32",
+        # },  # [dp(4)] + [ep(2) + moe_tp(2)]
+        # {
+        #     "tp_size": 1,
+        #     "pp_size": 1,
+        #     "ep_size": 4,
+        #     "overlap_communication": False,
+        #     "zero_stage": 0,
+        #     "precision": "fp32"
+        # },  # full dp for non-moe and full ep for moe
+    ],
+)
+def run_deepseek_test(test_config):
+    sub_model_zoo = model_zoo.get_sub_registry("transformers_deepseek")
+
+    for name, (model_fn, data_gen_fn, output_transform_fn, loss_fn, _) in sub_model_zoo.items():
+        check_forward_backward(model_fn, data_gen_fn, output_transform_fn, loss_fn, test_config)
+
+    clear_layout_converter()
+    Randomizer.reset_index()
+    torch.cuda.empty_cache()
+
+
+def check_deepseek(rank, world_size, port):
+    disable_existing_loggers()
+    colossalai.launch(rank=rank, world_size=world_size, host="localhost", port=port, backend="nccl")
+    run_deepseek_test()
+
+
+@pytest.mark.skip("redundant")
```

Comment:
As this is redundant, why do u add it 😂

Reply:
resolved in [`3d0a30a` (#5925)](https://github.com/hpcaitech/ColossalAI/pull/5925/commits/3d0a30a9bbab3fd20c55dd93b60f63b347348846)